tests/test_single_topology_v3.py

```diff
@@ -1,33 +1,40 @@
-# test that end-states are setup correctly in single topology calculations.
-import functools
 import multiprocessing
 import os
-from importlib import resources
-
-from timemachine.constants import DEFAULT_FF
-from timemachine.ff.handlers import openmm_deserializer
 
 os.environ["XLA_FLAGS"] = "--xla_force_host_platform_device_count=" + str(multiprocessing.cpu_count())
```

Comment:
Should we set this in the makefile? Maybe thats not why the imports changed, if so this is just a nit?

Reply:
Hm, not sure I understand. As far as I understand this needs to be set in the environment when jax is loaded at runtime.

This was not added in the PR; the change in the imports was just to combine two timemachine import sections that were previously separated by this line (so isort wouldn't automatically fix them).

Reply:
@badisa were you thinking to set this in the Makefile `*_tests` targets? I think that would make sense for CI, but would break running individual tests with `pytest -k`

Reply:
Perhaps at some point we should investigate `pytest-env` to set this in our pytest.ini. 

In the mean time perhaps just add a comment about not importing timemachine above the line? But not important to PR, so do at your discretion.